agreement-view-component.html

```diff
@@ -0,0 +1,18 @@
+<stackable-modal stackable="$ctrl.visible.agreementModal">
+  <br-modal br-title="{{$ctrl.agreementHeader}}">
+    <br-modal-body>
+        <br-agreement
+          ng-model="$ctrl.agreed"
+          br-agreement-group="{{$ctrl.group}}"
+          br-show-header="false">
+        </br-agreement>
+        <br><br>
```

Comment:
Adding extra vertical space.  For some reason the <br-agreement> component overruns into the modal footer.  If anyone knows how to fix the br-agreement template to prevent this, that would be good.